### ISSUE TITLE
Issue on the bash major version check

### ISSUE THREAD

**Issue by grational** [sdkman/sdkman-cli#649]
Hi sdkman-cli team, 

I've received a message from the `sdk` tool about my outdated version of bash. It seems strange to me since I'm using a recent version of Lubuntu (16.04). 

It seems a localization problem in the code inside `sdkman-utils.sh` file, line `108`:
```bash
-z "$(bash --version | grep 'GNU bash, version 4..*')"
```
Infact, the actual message given by my `bash --version` is slightly different because of the Italian localization:
```bash
GNU bash, versione 4.3.48(1)-release (x86_64-pc-linux-gnu)
```

**Comment by jjYBdx4IL** [comment]
confirmed, make sure to put  a "export LC_ALL=C" inside a bash script to make it independent of current environment. "export TZ=UTC" is usually a good idea, too.

**Comment by tmysik** [comment]
Same problem here, czech version.

**Comment by marc0der** [comment]
The fix has been merged and is available on the beta channel for testing. If I get the thumbs up I'll release it to stable.

**Comment by marc0der** [comment]
This was released as 5.6.4 and resolves the problem so closing this issue.
